lib/osf-components/addon/components/registries/finalize-registration-modal/component.ts

```diff
@@ -0,0 +1,43 @@
+import { tagName } from '@ember-decorators/component';
+import { computed } from '@ember-decorators/object';
+import Component from '@ember/component';
+import { action } from '@ember/object';
+import moment from 'moment';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import {
+    FinalizeRegistrationModalManager,
+} from 'osf-components/components/registries/finalize-registration-modal/manager/component';
+
+import styles from './styles';
+import template from './template';
+
+@layout(template, styles)
+@tagName('')
+export default class FinalizeRegisrationModalComponent extends Component {
+    manager!: FinalizeRegistrationModalManager;
+    groupValue: string = '';
+    embargoRangeStartDate: Date = moment().add(3, 'days').toDate();
+    embargoRangeEndDate: Date = moment().add(4, 'years').toDate();
```

Comment:
Please indicate which of these are required parameters, optional parameters, and private properties.